.packit.yaml

```diff
@@ -31,11 +31,11 @@ jobs:
   - job: copr_build
     trigger: pull_request
     targets:
-      centos-stream-8:
-        additional_modules: "foreman:el8"
+      rhel-9:
+        additional_modules: "foreman-devel:el9"
         additional_repos:
-          - http://koji.katello.org/releases/yum/foreman-nightly/el8/x86_64/
-          - http://yum.theforeman.org/plugins/nightly/el8/x86_64/
+          - https://yum.theforeman.org/releases/nightly/el9/x86_64/
+          - https://yum.theforeman.org/plugins/nightly/el9/x86_64/
     module_hotfixes: true
```

Comment:
I think we can drop this now

Reply:
Same answer as in the pulp PR.

Do we need it today? No.
Will it be useful in the future if we ever decide to flip Ruby to anything other than 3.0 and not have to touch every single packit yaml? Yes :)